### PR TITLE
Move Native Social SIWA to Native Connections page

### PR DESCRIPTION
**Pull request by tamigoodall**
Moved Native Social SIWA to Native connections list
Standardized and normalized titles for all social connection docs
Updated links.
Moved non-native set up steps to apple.md file
Deleted unused file

Review: https://docs-content-staging-pr-9036.herokuapp.com/docs/connections/identity-providers-social

### CODE REVIEW

**Comment by erjohnson** [comment]
May want to add frontmatter similar to facebook native so it renders in the list at the bottom of https://docs-content-staging-pr-9036.herokuapp.com/docs/connections/identity-providers-social

File: articles/connections/social/apple.md [hunk 1]
```diff
@@ -16,37 +17,58 @@ useCase:
...
 
 ::: warning
-The [Apple App Store Developer Guidelines](https://developer.apple.com/app-store/review/guidelines/#sign-in-with-apple) require that SIWA must be available in all applications that exclusively use third-party sign-in options, such as Facebook or Google.
+The [Apple App Store Developer Guidelines](https://developer.apple.com/app-store/review/guidelines/#sign-in-with-apple) require that Sign In with Apple (SIWA) must be available in all applications that exclusively use third-party sign-in options, such as Facebook or Google.
 :::
 
 ## Prerequisites
 
-Before you add support for SIWA to your app, you'll need:
+Before you configure SIWA for your native app in Auth0, you must:
```
Comment (rkhoriander) [inline articles/connections/social/apple.md]: This is supposed to be only for web apps, correct? Not sure why we added "native"?

File: articles/connections/social/apple.md [hunk 2]
```diff
@@ -16,37 +17,58 @@ useCase:
...
+
+* Have an [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/).)
```
Comment (rkhoriander) [inline articles/connections/social/apple.md:34]: Isn't the iOS program for development on native devices? If this is for web apps only, is it unnecessary?

**Review by rkhoriander — CHANGES_REQUESTED** [review]

File: articles/connections/social/apple.md (hunk 2 above)
Comment (jeffreylees) [reply articles/connections/social/apple.md:34]: It is definitely necessary. This program is how you gain access to set up, configure, and use SIWA with any app of any kind, unless they've added a way to do it with web apps without registering.

File: config/redirects.js [hunk 3]
```diff
@@ -2435,6 +2435,10 @@ module.exports = [
       from: '/connections/apple-setup',
       to: '/connections/apple-siwa/set-up-apple'
     },
+    {
+      from: '/connections/apple-siwa/add-siwa-web-app',
+      to: '/connections/social/apple'
+    },
```
Comment (jeffreylees) [inline config/redirects.js:2441]: Going to sound like a broken record here, but more than one page moved right? Redirects for any page that moves ever.

Comment (tamigoodall) [reply config/redirects.js:2441]: only one page moved

Comment (jeffreylees) [reply config/redirects.js:2441]: `/connections/apple-siwa/add-siwa-to-web-app` was deleted, then remade here `/connections/social/apple` (redirect exists)

`connections/apple-siwa/add-siwa-to-native-app`  was moved to `/connections/nativesocial/add-siwa-to-native-app` (no redirect)

Comment (rkhoriander) [reply config/redirects.js:2441]: I added it.

**Review by rkhoriander — APPROVED** [review]

**Pull request merged**